# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libwpd/hotfix-size_t.patch.libwpd
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libwpd-0.10.3/src/lib/WPXTable.h.vanilla	2021-12-13 18:05:20.332002174 +0100
+++ libwpd-0.10.3/src/lib/WPXTable.h	2021-12-13 18:06:17.008002201 +0100
@@ -36,6 +36,7 @@
 #ifndef _WPXTABLE_H
 #define _WPXTABLE_H
 
+#include <stddef.h>
 #include <vector>
 
 struct WPXTableCell
